ppx/element_content.ml

```diff
@@ -181,6 +181,8 @@ let table ~lang ~loc ~name children =
   let thead, others = partition (html "thead") others in
   let tfoot, others = partition (html "tfoot") others in
 
+  let others = filter_whitespace others in
```

Comment:
That's going to filter whitespaces everywhere else, that's not correct.

Reply:
I though it would remove whitespaces around `tr`.
`others` should contains only `tr` ?

Reply:
Ah, you are right, I misread the code. In this case, should't whitespace be filtered directly at the beginning, before the partitions ?

Reply:
In that case, with the filter_whitespace annotation !